test/crawler/keyword/keywords_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Crawler.Keyword.KeywordsTest do
+  use Crawler.DataCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.Keyword
+
+  describe("list_keywords/1") do
+    test "given a valid user id, returns user's keyword list" do
+      %{id: user_id} = insert(:user)
+      keyword = insert(:keyword, user_id: user_id)
+
+      assert Keywords.list_keywords(user_id) == [keyword]
+    end
+  end
+
+  describe("create_keyword/1") do
+    test "given a valid keyword data, returns a saved keyword" do
+      %{id: user_id} = insert(:user)
+      keyword_params = %{name: "keyword", user_id: user_id}
+
+      assert {:ok, %Keyword{} = keyword} = Keywords.create_keyword(keyword_params)
+      assert keyword.name == "keyword"
```

Comment:
should we test `keyword.user_id == user_id` :D

Reply:
Fix in [e30ade7](https://github.com/vnntsu/elixir-ic/pull/36/commits/e30ade7853ad15da4e2ca9ca7f92f5a63f2ee6df)